lib/players/human.rb

```diff
@@ -0,0 +1,13 @@
+require 'pry'
+
+module Players
+
+  class Human < Player
+    attr_accessor :input
+
+    def move(game)
+      @input = gets.strip
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  gets.strip
end
```